server/reports/requisition/2_5_0/src/template.html

```diff
@@ -61,7 +62,7 @@
         <td class="item_name" style="width: 25%">{{ line.item.name }}</td>
         <td class="quantity" style="width: 8%">{{ line.itemStats.availableStockOnHand }}</td>
         <td class="pack" style="width: 10%">{{ line.itemStats.averageMonthlyConsumption | round(precision=2) }}</td>
-        <td class="batch" style="width: 8%">
+        <td class="stock_to_order" style="width: 8%">
           {{ data.requisition.maxMonthsOfStock * line.itemStats.averageMonthlyConsumption }}
         </td>
```

Comment:
Is it supposed to be obvious what "stock to order" means? This looks like its calculating something more like max stock?

Reply:
also should the column be rounding? 

![Screenshot 2025-01-22 at 1 43 13 PM](https://github.com/user-attachments/assets/ae741e0d-5857-4ad5-a02e-4bf0516b0ad7)

Reply:
Actually it doesn't make sense because it's basically suggested but without taking away stock on hand.. I'm not sure what the original intention of this report was 🤔 it's quite an old report... maybe we should just remove the column?

Reply:
Probs.. whats OG report do?

@mark-prins ?

Reply:
OG doesn't have

Reply:
ah... i reckon take it out then - all the other important columns are there?

Reply:
yeah

Reply:
I've removed, and added MOS to match with OG report